projects/go-lib/src/public_api.ts

```diff
@@ -96,11 +96,15 @@ export * from './lib/components/go-modal/go-modal.service';
 export * from './lib/components/go-off-canvas/go-off-canvas.component';
 export * from './lib/components/go-off-canvas/go-off-canvas.module';
 export * from './lib/components/go-off-canvas/go-off-canvas.service';
+export * from './lib/components/go-off-canvas/go-off-canvas.interface';
 
 // Pill
 export * from './lib/components/go-pill/go-pill.component';
 export * from './lib/components/go-pill/go-pill.module';
 
+// Portal
+export * from './lib/components/go-portal/go-portal.module';
```

Comment:
Should we also export the `portalService` here?

Reply:
I don't think so. The api of this module shouldn't require you to ever need the service.

Reply:
Ah, yeah I guess you're right.